### PR TITLE
Change corpus used by LSA primitive

### PR DESCRIPTION
**Pull request by dsherry**
Change from `brown` to `gutenberg` as `gutenberg` is public-domain.

https://www.nltk.org/nltk_data/

### CODE REVIEW

**Review by eccabay — APPROVED** [review]
LGTM! I agree with Gaurav's suggestion to add `corpus` as a parameter, but I don't think that's blocking.

File: nlp_primitives/lsa.py [hunk 1]
```diff
@@ -53,9 +53,9 @@ def __init__(self):
         self.number_output_features = 2
         self.n = 2
 
-        brown = nltk.corpus.brown.sents()
+        gutenberg = nltk.corpus.gutenberg.sents()
```
[the remark this answers is not in the document]
Comment (dsherry) [reply nlp_primitives/lsa.py:56]: @gsheni yep that would be awesome. We have a TODO for this 3 lines above! Fundamentally, I don't think featuretools primitives can be parameterized today.

[the remark this answers is not in the document]
Comment (gsheni) [reply nlp_primitives/lsa.py:56]: Yes, Featuretools can be parameterized today. Its something we've supported for a while.
You can have a primitive with no arguments, default parameters, or required parameters. 
```python
time_since_previous = TimeSincePrevious(unit="minutes")
ft.df(transform_primitives=[time_since_previous]...)
```

[the remark this answers is not in the document]
Comment (dsherry) [reply nlp_primitives/lsa.py:56]: @gsheni ah, very cool! My bad, I think I was getting this confused with having dfs optionally "fit" primitives before using them to transform, which was a separate conversation that's been going around.

If you wanna file an issue to make the TF-IDF corpus parameterizable in the LSA primitive, please do :)

**Pull request merged**